bulk_adding/views.py

```diff
@@ -199,6 +201,7 @@ def update_person(self, context, data, person_extra):
                 extra__election=election,
                 role=election.candidate_membership_role,
             ):
+            raise_if_unsafe_to_delete(old_membership)
```

Comment:
I think this can't leave anything in an inconsistent state since `update_person` is only called from within an `transaction.atomic()` block.